src/java/io/bazel/rulesscala/scalac/StreamReporter.java

```diff
@@ -0,0 +1,24 @@
+package io.bazel.rulesscala.scalac;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import scala.tools.nsc.Settings;
+import scala.tools.nsc.reporters.ConsoleReporter;
+
+class StreamReporter extends ConsoleReporter {
+  public StreamReporter(Settings settings, PrintStream out, PrintStream err) {
+    super(
+        settings,
+        new BufferedReader(new InputStreamReader(new ByteArrayInputStream(new byte[0]))),
```

Comment:
why do we need to buffer an empty input stream?

Reply:
[The inherited constructor from ConsoleReporter expects a BufferedReader ](https://github.com/scala/scala/blob/8a2cf63ee5bad8c8c054f76464de0e10226516a0/src/compiler/scala/tools/nsc/reporters/ConsoleReporter.scala#L21)